components/AudioPlayer.module.css

```diff
@@ -0,0 +1,198 @@
+.audio_player_section_container {
+  position: relative;
+  width: 100%;
+  height: 100vh;
+  padding: 10%;
+}
+
+.image_wrapper {
+  position: relative;
+  top: 30px;
+  width: 180px;
+  height: 180px;
+  margin: 0 auto;
+  box-shadow: 0 0 20px black;
+}
+
+.episode_info_section{
+  position: relative;
+  top: 50px;
+  display: block;
+  text-align: center;
+  margin: 0 auto;
+  padding: 0 20px;
+}
+
+.episode_title {
+  text-align: center;
+}
+
+.episode_details{
+  width: 100%;
+  height: 150px;
+  overflow: scroll;
+  text-align: start;
+}
+
+
+.audioPlayer {
+    --primary: grey;
+    --secondary: white;
+    position: absolute;
+    bottom: 10%;
+    width: 80%;
+    text-align: center;
+  }
+
+  .audioPlayerPlaybutton{
+    display: flex;
+    justify-content: center;
+    gap: 20px;
+    margin: 20px;
+  }
+  
+  .forwardBackward {
+    background: none;
+    border: none;
+    display: flex;
+    align-items: center;
+    font-family: var(--title-font);
+    font-size: 12px;
+    cursor: pointer;
+  }
+  
```

Comment:
```suggestion
    position: relative;
  }
  
  .forwardBackward svg {
	position: relative;
	top: 0.2rem;
}
```
![スクリーンショット 2021-12-13 0 05 55](https://user-images.githubusercontent.com/51708229/145774711-c2adac90-073d-4e46-bdd8-8f811b5f2edb.png)

Even though having `align-items: center`, the arrow and "30" align a bit off, so we can adjust it like this.
I haven't checked every device so it might be off again in the different devices.